.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test on ${{ matrix.os }} with Rust ${{ matrix.rust }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        rust:
+          - 1.21.0
+          - stable
+          - beta
+          - nightly
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Install Rust ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
```

Comment:
I think `rustup update && rustup default` should work ok here, I don't think there's any need to use `actions-rs/toolchain`

Reply:
I rebased this PR onto your changes in master, so this should be resolved now.